db/define_schema.rb

```diff
@@ -11,4 +11,29 @@
 # Do not define those columns yourself.
 
 # What table do you need to store the necessary information?
-# What columns do you need in that table?
\ No newline at end of file
+# What columns do you need in that table?
+DB.define_table("swimmers")
+DB.define_column("swimmers","swimmer_id","integer")
+DB.define_column("swimmers","name","string")
+DB.define_column("swimmers","school_id","integer")
+
+DB.define_table("schools")
+DB.define_column("schools","school_id","integer")
+DB.define_column("schools","school_name","string")
+DB.define_column("schools","conference_id","integer")
+
+DB.define_table("events")
+DB.define_column("events","event_id","integer")
```

Comment:
You should not be defining each table's `id` column. ActiveRecord does that for you.